cardano-db/src/Cardano/Db/Types.hs

```diff
@@ -133,3 +140,35 @@ showRewardType rt =
 word64ToAda :: Word64 -> Ada
 word64ToAda w =
   Ada (fromIntegral w / 1000000)
+
+-- | The stake pool identifier. It is the hash of the stake pool operator's
+-- vkey.
+--
+-- It may be rendered as hex or as bech32 using the @pool@ prefix.
+newtype DbPoolId = DbPoolId { getPoolId :: Text }
```

Comment:
* In the exisiting `db-sync` code **all** database fields the have a suffix of `Id` are postgres indices. That means we should avoid using the `Id` suffix for anything that is not a postgres index.
* Is that field a hash? If so, why is it `Text` and not `ByteString`? Please tell me you are not storing raw `ByteString`s in the database hex encoded.